rasa/nlu/featurizers/sparse_featurizer/lexical_syntactic_featurizer.py

```diff
@@ -166,20 +168,24 @@ def _create_sparse_features(self, message: Message) -> None:
         features."""
         import scipy.sparse
 
-        tokens = message.get(TOKENS_NAMES[TEXT])
+        # TODO: do I get it right that these features will still exist only for user text?
+        tokens = None
+        if not message.get(MESSAGE_ACTION_NAME):
```

Comment:
yes, but why do we need to check for action_text, doesn't it go to separate message attribute?